content/lua/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: 'Conditionals' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Conditionals take an expression and evaluate it to `true` or `false`. Code blocks run depending on the outcome of the expression.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Conditionals'
+  - 'Control FLow'
+  - 'Operators'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+**Conditionals** take an expression and evaluate if it’s `true` or `false`. Code blocks run depending on the outcome of the expression. 
+
+The condition can contain logic [operators](https://www.codecademy.com/resources/docs/lua/operators) and [variables](https://www.codecademy.com/resources/docs/lua/variables). The condition can be evaluated as `true` or `false`.
+
+
+![A diagram explaining a control flow based on the validity of different statements.](https://content.codecademy.com/courses/learn-python-control-flow/control-flow.svg)
+## If Statements
+```lua
```

Comment:
```suggestion
```pseudo
```

Reply:
Just to clarify the reason behind paul's comment here, since `<condition>` and `<code>` aren't part of the Lua language syntax, even though its structured that way, we should used the `pseudo` codeblock styling.